mappers/bluetooth_mapper/controller/controller.go

```diff
@@ -86,7 +86,8 @@ func (c *Config) Start() {
 
 	for _, schedule := range c.Scheduler.Schedules {
 		helper.ControllerWg.Add(1)
-		go schedule.ExecuteSchedule(c.ActionManager.Actions, c.Converter.DataRead, c.Device.ID)
+		newSchedule := schedule
+		go newSchedule.ExecuteSchedule(c.ActionManager.Actions, c.Converter.DataRead, c.Device.ID)
```

Comment:
Good catch, and if we don't declare the newSchedule here, will the last schedule be executed in multi goroutine?

Reply:
@markwangC I write an example, and it seems that not the last iterator has only been executed, PTAL https://play.golang.org/p/jq8jEUyFe0d